include/opw_kinematics/opw_parameters_examples.h

```diff
@@ -2,15 +2,8 @@
 #define OPW_PARAMETER_EXAMPLES_H
 
 #include "opw_kinematics/opw_kinematics.h"
+#include <opw_kinematics/opw_macros.h>
```

Comment:
Would we perhaps want to `#include` the file in the `.cpp`? It doesn't appear to be used in this file.